api/client/stats.go

```diff
@@ -71,8 +71,12 @@ func (s *containerStats) Collect(cli *DockerCli, streamStats bool) {
 			s.Memory = float64(v.MemoryStats.Usage)
 			s.MemoryLimit = float64(v.MemoryStats.Limit)
 			s.MemoryPercentage = memPercent
-			s.NetworkRx = float64(v.Network.RxBytes)
-			s.NetworkTx = float64(v.Network.TxBytes)
+			// Note: The cgroup network Rx and Tx are flipped compared to ifconfig
```

Comment:
@ottok I might be wrong here and not be fully aware of the context, but please check the changes in #14324 and see if it is still needed to swap rx/tx counters. 
Docker 1.7.1 now uses libnetwork to retrieve the container interface stats and libnetwork reports them as they would appear in the ifconfig o/p on the container.